releases/8.1/languages/en.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+return [
+    'common_header' => 'PHP 8.1 is a major update of the PHP language. enums, readonly properties, first-class callable syntax, fibers, intersection types, performance improvements and more.',
+    'main_title' => 'Released!',
+    'main_subtitle' => 'PHP 8.1 is a major update of the PHP language.<br class="display-none-md"> It contains many new features and optimizations including enums, readonly&nbsp;properties, first-class&nbsp;callable&nbsp;syntax, fibers, intersection&nbsp;types, performance improvements and more.',
```

Comment:
```suggestion
    'main_subtitle' => 'PHP 8.1 is a major update of the PHP language.<br class="display-none-md"> It contains many new features, including enums, readonly&nbsp;properties, first-class&nbsp;callable&nbsp;syntax, fibers, intersection&nbsp;types, performance improvements and more.',
```
I found the "and optimizations" a bit awkward here, as the following is a list of features and we have "performance improvements" at the end.